### PR TITLE
Various updates to improve test coverage

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Updates to improve test coverage with changes made for implementing Koalas support.

### CODE REVIEW

File: featuretools/tests/computational_backend/test_calculate_feature_matrix.py [hunk 1]
```diff
@@ -1138,12 +1137,13 @@ def test_cfm_returns_original_time_indexes(es):
...
-        # Dask doesn't return time (doesn't support multi-index)
-        # Koalas does support multiindex, look into possibly supporting
-        time_level_vals = []
+    # else:
+    #     # Code needed if removing dask/koalas xfail
+    #     fm = to_pandas(fm, index='id')
+    #     instance_level_vals = fm.index
+    #     # Dask doesn't return time (doesn't support multi-index)
+    #     # Koalas does support multiindex, look into possibly supporting
+    #     time_level_vals = []
```
Comment (thehomebrewnerd) [inline featuretools/tests/computational_backend/test_calculate_feature_matrix.py:1146]: Maybe we should just remove this for now as it isn't needed?

Comment (frances-h) [reply featuretools/tests/computational_backend/test_calculate_feature_matrix.py:1146]: I think it makes sense to get rid of the extra code/comments for x-failed tests, it's probably better to keep the tests clean and uncluttered with unused code.

Comment (thehomebrewnerd) [reply featuretools/tests/computational_backend/test_calculate_feature_matrix.py:1146]: That's my thought as well. I'll remove this and look for any other unused blocks that can be cleaned up.

**Comment by thehomebrewnerd** [comment]
@frances-h Didn't know if you wanted to take another look over this before I merge it in to `koalas-entity`.

**Pull request merged**